expo/gbe/models.py

```diff
@@ -395,21 +395,23 @@ def sched_payload(self):
 
     def has_role_in_event(self, role, event):
         '''
-        Gets all of a person's roles for a conference
+        Returns True if this person has the
+        given role in the given event
         '''
         doing_it = False
         if role == "Performer":
             for show in self.get_shows():
                 if show.pk == event.pk:
-                    doing_it = (doing_it or True)
-        else:
-            event_workers = event.roles(role)
-            if self in event_workers:
-                doing_it = True
-            else:
-                for perf in self.get_performers():
-                    doing_it = doing_it or (perf in event_workers)
-
+                    doing_it = True
```

Comment:
I see the jury is now out on the preference online (this was an interesting read:  http://stackoverflow.com/questions/36707/should-a-function-have-only-one-return-statement)

... but back when I was originally taught to code, my teachers were fairly stringent on the "one return statement per function" approach and I'm generally in the habit of it.

Having had to debug the opposite of that a number of times in different codebases and ending up feeling like a goalie with a ball that shoots between my legs when the code dumps out of the function unexpectedly, I tend to strongly prefer the 1 return statement approach in the case where the logic gets more complicated down the line.